Day6/part_five.js

```diff
@@ -0,0 +1,19 @@
+// /*5.Use loop to print the following pattern:
+// 0 x 0 = 0
+// 1 x 1 = 1
+// 2 x 2 = 4
+// 3 x 3 = 9
+// 4 x 4 = 16
+// 5 x 5 = 25
+// 6 x 6 = 36
+// 7 x 7 = 49
+// 8 x 8 = 64
+// 9 x 9 = 81
+// 10 x 10 = 100
+// */
+function printMultiplication(start, end) {
+  for (let i = start; i <= end; i++) {
+    console.log(`${i} * ${i} = ${i * i}`);
```

Comment:
it seems more similar whit cross
```suggestion
    console.log(`${i} x ${i} = ${i * i}`);
```